### ISSUE TITLE
[Windows Terminal]: Screen Reader is not announcing information pop-up message when user open terminal app. 

### ISSUE THREAD

**Issue by v-rpundir** [microsoft/terminal#11929]
### Windows Terminal version

1.12.2931.0

### Windows build number

10.0.22504.1010

### Other Software

Test Environment:
OS: Windows 11 Version Dev (OS Build 22504.1010)
App: Windows Terminal Preview
Screen Reader: Narrator
Tool: Accessibility Insights for Windows Version 1.1.1741.1

### Steps to reproduce

**Pre-requisites: **
1. Open Terminal then open Settings page using 'Ctrl+,'
2. Inside Startup page, Set Default Terminal Application as 'Windows Console Host'.
3. Save changes and close Terminal application.

**Repro Steps: **
1. Open Narrator using 'Win + Ctrl + Enter' key.
3. Open Terminal app.
4. Verify that Narrator is announcing information pop-up message or not.

**User Experience: **
Screen Reader users will be impacted here as they are not getting information which appears on the pop-up. As a result, they will remain unaware about the information.

**Guideline Reference: **
https://www.w3.org/WAI/WCAG21/Understanding/status-messages

**Attachments:**
[Screen Reader is not annoucning information pop-up message.....zip](https://github.com/microsoft/terminal/files/7701158/Screen.Reader.is.not.annoucning.information.pop-up.message.zip)

### Expected Behavior

Screen Reader should announce information pop-up message when it appear on the screen. Screen Reader should announce 'Windows Terminal can be setup as the default terminal application in your settings. Open Settings button Close button' 

### Actual Behavior

Screen Reader is not announcing information pop-up message when it appears on the screen.

Note Issue: Screen Reader is not announcing control name (Don't show again and X) of information pop-up when Screen Reader announce 'Termination behavior can be configured at advance profile settings'. (refer attachment 'Note Issue')
